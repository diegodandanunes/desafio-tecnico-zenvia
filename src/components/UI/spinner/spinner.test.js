import React from 'react';
import { render } from '@testing-library/react';

import { Spinner } from './';

it("renders", () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment()).toMatchSnapshot();
});
