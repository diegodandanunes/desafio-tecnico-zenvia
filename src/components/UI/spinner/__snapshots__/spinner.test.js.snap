// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm FlHQP"
  >
    <div
      class="lds-dual-ring"
    />
  </div>
</DocumentFragment>
`;
